presto-main/src/test/java/io/prestosql/security/TestAccessControlManager.java

```diff
@@ -378,6 +406,57 @@ public void testAllowExecuteFunction()
                 });
     }
 
+    private AccessControlManager createAccessControlManage(TestingEventListenerManager eventListenerManager, List<String> systemAccessControlProperties)
+            throws IOException
+    {
+        Path systemAccessControlConfig = createTempFile("access-manager-config-file", ".properties");
+        Files.write(systemAccessControlConfig, systemAccessControlProperties, TRUNCATE_EXISTING, CREATE, WRITE);
+
+        return createAccessControlManager(eventListenerManager, new AccessControlConfig()
+                .setAccessControlFiles(systemAccessControlConfig.toFile()
+                        .getAbsolutePath()));
+    }
+
+    private AccessControlManager createAccessControlManager(TransactionManager testTransactionManager)
+    {
+        return new AccessControlManager(testTransactionManager, emptyConfigManager(), new AccessControlConfig());
+    }
+
+    private AccessControlManager createAccessControlManager(EventListenerManager eventListenerManager, AccessControlConfig config)
+    {
+        return new AccessControlManager(createTestTransactionManager(), eventListenerManager, config);
+    }
+
+    private SystemAccessControlFactory eventListeningSystemAccessControlFactory(String name, EventListener eventListener)
+    {
+        return new SystemAccessControlFactory()
+        {
+            @Override
+            public String getName()
+            {
+                return name;
+            }
+
+            @Override
+            public SystemAccessControl create(Map<String, String> config)
+            {
+                return new SystemAccessControl()
+                {
+                    @Override
+                    public void checkCanSetSystemSessionProperty(SystemSecurityContext context, String propertyName)
```

Comment:
Why you had to implement this?

Reply:
Because it does not have its default implementation, maybe it should?

Reply:
I think it should like everything else.

Reply:
hm, so maybe in different commit in same pr? or a separate pr?

Reply:
> different commit in same pr

👍 

Actually, both ways are good to me. Up to you.